examples/migraphx_driver/README.md

```diff
@@ -0,0 +1,604 @@
+# MIGraphX Driver
+
+## Description
+The MIGraphX driver is a tool that allows you to utilize many of the core functions of MIGraphX without having to write your own program. 
+
+## How to Use this Example
+
+The MIGraphX driver is built alongside MIGraphX and can be found in `/opt/rocm/bin/migraphx-driver`
```

Comment:
Can you add "or in the folder "build/bin" after building the source code?"

Reply:
Added